src/app/modules/submission/encrypt-submission/encrypt-submission.controller.ts

```diff
@@ -57,11 +58,28 @@ const EncryptSubmission = getEncryptSubmissionModel(mongoose)
 // NOTE: Refer to this for documentation: https://github.com/sideway/joi-date/blob/master/API.md
 const Joi = BaseJoi.extend(JoiDate)
 
-const submitEncryptModeForm: RequestHandler = async (req, res) => {
+const submitEncryptModeForm: RequestHandler<
+  { formId: string },
+  {
+    message: string
+    submissionId?: string
+    spcpSubmissionFailure?: boolean
+    isPageFound?: boolean
+    formTitle?: string
```

Comment:
I think you can remove `isPageFound` and `formTitle`. they're not used by the frontend.

Reply:
I think it is still used in some bits of the code right? https://github.com/opengovsg/FormSG/blob/aecc1dbd44b4258584819a2b3e822347c1446be9/src/public/modules/forms/services/form-error.client.factory.js#L40-L44

Reply:
yup they're used elsewhere, but not in this particular case of when storage mode submissions failing.